ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -8282,6 +8282,20 @@ keys:
                   - dict
                   items:
                     type: str
+                access_lists:
```

Comment:
we should remove `primary_key` line 8269 above

Reply:
Not possible because of conversion from wildcard dict, but I have added a note with TODO in the schema.